.github/workflows/android_build.yml

```
@@ -14,10 +14,10 @@ jobs:
     steps:
 
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: set up JDK 1.8
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v4
```

actionlint (1.7.4)

missing input "distribution" which is required by action "actions/setup-java@v4". all required inputs are "distribution"

(action)